auth/gcloud/aio/auth/auth.py

```diff
@@ -191,3 +192,60 @@ def __init__(self, service_file: Optional[str] = None,
         self.access_token_duration = int(content['expires_in'])
         self.access_token_acquired_at = datetime.datetime.now()
         self.acquiring = None
+
+
+SCOPES = ['https://www.googleapis.com/auth/cloud-platform']
+
+
+class IamCredentialsClient:
+    def __init__(self, project: Optional[str] = None,
+                 service_file: Optional[str] = None,
+                 session: Optional[aiohttp.ClientSession] = None,
+                 token: Optional[Token] = None) -> None:
+        self.project = project
+        self.session = session
+        self.token = token or Token(service_file=service_file, session=session, scopes=SCOPES)
+
+    async def headers(self):
+        token = await self.token.get()
+        return {
+            'Authorization': f'Bearer {token}',
+        }
+
+    def service_account_email(self):
+        return self.token.service_data.get('client_email')
+
+    async def sign_blob(self, payload: str, service_account_email: Optional[str],
```

Comment:
Given you've marked `service_account_email` as `Optional`, should we give it a default `= None`?

Reply:
Done